tests/integration_tests/performance/test_network_latency.py

```diff
@@ -0,0 +1,202 @@
+# Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+# SPDX-License-Identifier: Apache-2.0
+"""Tests the network latency of a Firecracker guest."""
+
+import logging
+import platform
+import re
+import pytest
+import host_tools.network as net_tools
+from conftest import _test_images_s3_bucket
```

Comment:
We should not import symbols starting with `_` as they're designed for internal use within the file where they are declared.

Reply:
Done.